packages/components/src/FormField/FormField.css

```diff
@@ -215,10 +215,16 @@
   flex-direction: column;
 }
 
+/* If there's a label, add padding-top to wrapper to account for "absolute" positioned label */
+.inputWrapper:has(.label) .childrenWrapper {
```

Comment:
so we're removing the padding-top from `.input` and then re-adding it to `.childrenWrapper` I'd prefer we try to put the padding back on the same element if possible. makes things easier to follow and track down imo

<img width="577" alt="image" src="https://github.com/user-attachments/assets/34699bd1-5604-42f3-a6fa-36fdc8fa938f">
<img width="577" alt="image" src="https://github.com/user-attachments/assets/92c45e17-55b5-4f66-8de2-866fa151e4ea">

vs how it works on current master

<img width="572" alt="image" src="https://github.com/user-attachments/assets/d672cdd7-037d-4e39-8861-e016128c66ab">